_posts/2019-01-31-CocoaPods-1.7.0-beta.markdown

```diff
@@ -0,0 +1,205 @@
+
+App specs progress forward the concept of 'Isolated Development' in which a `podspec` can be used as the single piece of information needed to generated an entire (throwable) workspace to develop on.
+
+Here's an example of an app spec declaration:
+
+```ruby
+Pod::Spec.new do |s|
+  s.name         = 'CoconutLib'
+  s.version      = '1.0'
+  s.authors      = 'Coconut Corp', { 'Monkey Boy' => 'monkey@coconut-corp.local' }
+  s.homepage     = 'http://coconut-corp.local/coconut-lib.html'
+  s.summary      = 'Coconuts For the Win.'
+  s.description  = 'All the Coconuts'
+  s.source       = { :git => 'http://coconut-corp.local/coconut-lib.git', :tag => 'v1.0' }
+  s.license      = {
+    :type => 'MIT',
+    :file => 'LICENSE',
+    :text => 'Permission is hereby granted ...'
+  }
+
+  s.source_files = 'Sources/*.{h,m}'
+
+  s.app_spec 'SampleApp' do |app_spec|
+    app_spec.source_files = 'Sample/*.swift'
+  end  
+end
+```
+
+App specs can leverage the majority of the CocoaPods spec DSL to declare dependencies, build configuration settings, etc. 
+
+We hope in the future to promote app specs to a top-level concept in which consumer applications are described as an app spec completely eliminating the need of an `.xcodeproj` in order to integrate with CocoaPods.
+
+### Multiple Xcodeproj Generation
+
+Historically, CocoaPods has always generated a single `Pods.xcodeproj` that contains all of the required targets and build settings for your project to compile. Using just one project that incorporates your entire `Podfile` works just fine for smaller projects; however, as your project grows, so will the size of your `Pods.xcodeproj` file.
+
+The larger the `Pods.xcodeproj` file, the longer Xcode has to spend parsing its contents, which we have discovered leads to a degraded Xcode experience. Instead of putting all of your targets into one monolithic Xcode project, we’ve noticed some considerable performance improvements for larger CocoaPods projects by integrating each pod as its own separate Xcode project.
+
+Whether performance is a problem, or you just prefer setting up your projects using multiple Xcode projects, CocoaPods now supports this setting using the `generate_multiple_pod_projects` installation option.
+
+You can enable this in your `Podfile` like so:
+
+```ruby
+install! ‘cocoapods’, generate_multiple_pod_projects: true
+```
+
+By default this option is turned off but we encourage you to try it out and report any issues you find to us. We expect that going forward in a future major version update to CocoaPods this becomes the default option when it comes to integration.
+
+### Incremental Installation
+
+Instead of regenerating your entire workspace every time you run `pod install`, CocoaPods now supports only regenerating the pod targets that have changed since your previous installation. Depending on the size of your project, this can save anywhere from a couple seconds to minutes off every `pod install` invocation.
+
```

Comment:
Maybe add a note about CI here as well: `On CI environments that support caching between builds, this can leveraged to improve build times by adding "[DIRECTORY]" to your CI environment's cache` (I can't remember the cache location of the top of my head, is it in the sandbox?)